src/demo/README.md

```diff
@@ -0,0 +1,58 @@
+## Prepare
+### Run docker-compose
+   It'll start:
+   - Postgres db for the VCAuthn service
+   - Keycloak with a pre-configured vc-authn realm
+
+#### Start VCAuthn service
+    `dotnet run`
+
+#### Configure Presentation configuration
+```
+POST https://localhost:5001/api/vc-configs
+Content-Type: application/json
+
+{
+    "id" : "test",
+    "subject_identifier" : "attribute1",
+    "configuration" : {
+        "name" : "test",
+        "version" : 1.0,
+        "requested_attributes" : {
+            "attribute1": {
+                "name" : "attribute1",
+                "restrictions" : [
+                    {
+                        "schema_id": "123",
+                        "schema_issuer_did": "",
+                        "schema_name": "name",
+                        "schema_version": "",
+                        "issuer_did": "",
+                        "cred_def_id": "",
+                    }
+				]
+            },
+            "attribute2": {
+                "name" : "attribute2",
+                "restrictions" : [
+                    {
+                        "schema_id": "345",
+                        "schema_issuer_did": "",
+                        "schema_name": "name",
+                        "schema_version": "",
+                        "issuer_did": "",
+                        "cred_def_id": "",
+                    }
+				]
+            }
+		}
+    }
+}
+```
+
+## Test
+#### Start Workflow
```

Comment:
"###" vs "####"?

Reply:
actually, you were right about `###` vs `####` all along 👍

Reply:
first time this week :)